src/answers-headless.ts

```diff
@@ -27,108 +27,247 @@ import answersUtilities from './answers-utilities';
 import { SelectableFilter } from './models/utils/selectablefilter';
 import { transformFiltersToCoreFormat } from './utils/transform-filters';
 
+/**
+ * Provides methods for updating the state and for sending requests to the Answers
+ * API through {@link AnswersCore}.
```

Comment:
I don' t think consumers of Headless need to know about Answers Core. What do you think about something like: "Provides the functionality for interacting with an Answers Search experience"

Reply:
That makes sense. I'll change it to that